docs/installing.md

```
@@ -0,0 +1,186 @@
+release](https://download.ledfx.app).
+
+There is no installation required, the application is portable.
+
+## Linux Installation
+
+To install on Linux first ensure you have at least Python 3.9 installed.
+
+1.  Install LedFx and all the dependencies using pipx:
+
+    ``` console
+    $ python -m pip install ledfx
```

Markdownlint (0.35.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)